_posts/2019-08-08-text-and-typography.md

```diff
@@ -101,6 +101,10 @@ Click the hook will locate the footnote[^footnote], and here is another footnote
 ![Desktop View](https://cdn.jsdelivr.net/gh/cotes2020/chirpy-images/posts/20190808/mockup.png)
 _Full screen width and center alignment_
 
+- Lightbox (with caption)
+{% lightbox https://cdn.jsdelivr.net/gh/cotes2020/chirpy-images/posts/20190808/mockup.png --thumb="https://cdn.jsdelivr.net/gh/cotes2020/chirpy-images/posts/20190808/mockup.png" --data="appfoundry_image_set" --title="Image opening in a lightbox" --alt="This is my image" --img-style="max-width:80%;" --class="yourclass" %}
```

Comment:
Using Jekyll `tag` will make it difficult to insert pictures, maybe it would be better to use JS/CSS to implement this function.

Anyway, thanks for submitting this PR.